source/PlayState.hx

```diff
@@ -66,11 +72,32 @@ class PlayState extends FlxState
 		add(deadlyObstacleGenerator.obstacles);
 	}
 
+	private function setUpTokens()
+		{
+			var generatedTokens = new FlxTypedGroup<Token>();
+			for (i in 0...10)
+			{
+				var token = new Token();
+
+				token.kill();
+				generatedTokens.add(token);
+			}
+	
+			var baseTokenParameters = new ObstacleParameters(FlxG.width, FlxG.height, 100, 20, 20);
+			var tokenVariation = new ObstacleVariation(-1, 1, 0, 0);
+	
+			tokenGenerator = new ObstacleGenerator<Token>(SECONDS_PER_TOKEN,
+			baseTokenParameters, tokenVariation, generatedTokens);
+			add(tokenGenerator.obstacles);
+		}
```

Comment:
The indentation looks a little funky here in GitHub, maybe give it a quick once-over on your end to see if there's some unnecessary indentation that needs removing